src/components/molecules/DialogWithHeader.tsx

```diff
@@ -11,18 +11,28 @@ interface IProps {
   fullWidth?: boolean;
 }
 const useStyles = makeStyles({
+  dialog: {
+    fontFamily: fontFamilyString,
+    width: '600px',
+  },
   bar: {
-    height: 'inherit',
     display: 'flex',
-    flexDirection: 'row',
     width: 'inherit',
-    color: highlightDarkColor,
+    color: primaryColor,
     justifyContent: 'space-between',
     alignItems: 'center',
+    fontSize: '17px',
   },
   close: {
     color: closeButtonColor,
   },
+  content: {
+    padding: '22px 28px',
+  },
+  '.about-content': {
```

Comment:
Dialog Component should be agnostic to it's content
Never let generic component contain classes of it's children.

Also, naming classes like this is not according to `makeStyles`.
This is probably because you try to  style inner children of this component.